File: Java/Aeroplane.java
class Aeroplane extends Airway
{
    private String flightNo;

    Aeroplane(int id, String name, String engineType, String flightNo)
    {
        super(id, name, engineType);
        this.flightNo = flightNo;
    }

    @Override
    public void displayDetails()
    {
        super.displayDetails();
        System.out.println("Flight Number   : " + flightNo);
    }

    @Override
    void startJourney()
    {
        System.out.println("Aeroplane Journey Started Successfully...");
    }
}

File: Python/aeroplane.py
from airway import Airway

class Aeroplane(Airway):

    def __init__(self, vehicle_id, vehicle_name, engine_type, flight_no):
        super().__init__(vehicle_id, vehicle_name, engine_type)
        self.flight_no = flight_no

    def display_details(self):
        super().display_details()
        print("Flight Number   :", self.flight_no)

    def start_journey(self):
        print("Aeroplane Journey Started Successfully...")